l10n/bundle.l10n.ja.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
One gap that exist in the new localization file is that there was no way to introduce a key:sting mapping for better readability. This is something we can investigate further, and follow up on with VSCode team if needed.